cfme/tests/configure/test_access_control.py

```diff
@@ -799,16 +800,18 @@ def test_superadmin_tenant_crud(request):
         * Update name of tenat
         * Delete tenant
     """
-    tenant = Tenant(
+    roottenant = appliance.collections.tenants.get_root_tenant()
+    tenant = roottenant.tenants.create(
```

Comment:
**REQUIRED:** There is actually no reason why we need to do this from ```roottenant.tenants``` here, we could have done it from ```appliance.tenants```. Had tenants been special, it may have been so.

Reply:
Changed